pkg/kubernetes/scripts/kube-client.js

```diff
@@ -1315,9 +1316,10 @@
                 if (meta) {
                     ex = null;
                     if (meta.name !== undefined) {
+                        var check_re = (resource.kind == "User") ? USER_NAME_RE : NAME_RE;
```

Comment:
In javascript this variable isn't limited to the `if`-scope, hence we declare it outside (where `meta` is declared) to avoid potential confusion.